sklearn/linear_model/tests/test_sag.py

```diff
@@ -951,7 +953,7 @@ def test_multinomial_loss_ground_truth():
     # n_samples, n_features, n_classes = 4, 2, 3
     n_classes = 3
     X = np.array([[1.1, 2.2], [2.2, -4.4], [3.3, -2.2], [1.1, 1.1]])
-    y = np.array([0, 1, 2, 0])
+    y = np.array([0, 1, 2, 0], dtype=np.float64)
```

Comment:
I guess this change is needed for `LinearModelLoss.loss_gradient` to be computed on class labels provided as integers, is this right?

If this is, does this introduce a breaking change?

Reply:
The new losses only accept `np.float32` and `np.float64`. This is no breaking change, as `LogistigRegression` converts to these types anyway, https://github.com/scikit-learn/scikit-learn/blob/b242b9dd200cbb3f60247e523adae43a303d8122/sklearn/linear_model/_logistic.py#L1503-L1515